prime-dotnet-webapi/ViewModels/SiteRegistration/OrganizationClaimViewModel.cs

```diff
@@ -0,0 +1,9 @@
+namespace Prime.ViewModels
+{
+    public class OrganizationClaimViewModel
+    {
+        public int PartyId { get; set; }
+        public string PEC { get; set; }
+        public string ClaimDetail { get; set; }
```

Comment:
We added fluent validator a few PRs ago and have been very slowly adding validation to our View Models that represent incoming user data. I'd love to see some basic validation here :)

`PrivacyOfficeViewModel` was the first one to use it, the pattern would look something like:
```C#
using FluentValidation;

namespace Prime.ViewModels
{
    public class OrganizationClaimViewModel
    {
        public int PartyId { get; set; }
        public string PEC { get; set; }
        public string ClaimDetail { get; set; }
    }

    public class OrganizationClaimValidator : AbstractValidator<OrganizationClaimViewModel>
    {
        public OrganizationClaimValidator()
        {
            RuleFor(x => x.PartyId).NotEmpty();
            RuleFor(x => x.PEC).NotEmpty();
            RuleFor(x => x.ClaimDetail).NotEmpty();
        }
    }
}
```
You don't even have to register the validator anywhere, the app automatically finds all classes that inherit from `AbstractValidator` in the assembly due to line 99 in `Startup.cs`:
```C#
.AddFluentValidation(options => options.RegisterValidatorsFromAssemblyContaining<PrivacyOfficeValidator>())
```